src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -105,20 +106,32 @@ protected override bool IsParentAnAttribute(SyntaxNode node)
         {
             // If everything classifies the same way, then just pick that classification.
             using var _ = PooledHashSet<ClassifiedSpan>.GetInstance(out var set);
+            var isStatic = false;
+            var staticSpan = new TextSpan();
```

Comment:
you shouldn't need to store this span.  it's redundant with the one stored in `set`.